lib/rust/ensogl/core/src/display/shape/primitive/def/primitive.rs

```diff
@@ -172,15 +172,11 @@ define_sdf_shapes! {
     }
 
     HalfPlane () {
-        return bound_sdf(position.y, bounding_box(0.0,0.0));
+        return bound_sdf(-position.y, bounding_box(0.0,0.0));
     }
 
-    /// Cuts the provided angle from a plane. The angle faces upwards, so the angle of PI is equal
```

Comment:
That's because the comment was entirely incorrect, but I don't think this entity requires documentation. `BottomHalfPlane` is, well, the bottom half of the plane.